packages/block-editor/src/store/selectors.js

```diff
@@ -2293,7 +2293,8 @@ const checkAllowListRecursive = ( blocks, allowedBlockTypes ) => {
 function getUnsyncedPatterns( state ) {
 	const reusableBlocks =
 		state?.settings?.__experimentalReusableBlocks ?? EMPTY_ARRAY;
-
+	const { patternCatogoriesMap: categories } =
```

Comment:
```suggestion
	const { patternCategoriesMap: categories } =
```

Typo. Which if fixed on the primary PR will need updating here.